saltproc/core

```diff
@@ -0,0 +1,105 @@
+set title "themperature coefficient of reactivity 900K for both"
```

Comment:
Does this need to be included? should it be called core or should it have a more meaningful name/location?